README.md

```diff
@@ -314,7 +318,7 @@ Load a specific MP3D or Gibson house: `examples/example.py --scene path/to/mp3d/
 
 Additional arguments to `example.py` are provided to change the sensor configuration, print statistics of the semantic annotations in a scene, compute action-space shortest path trajectories, and set other useful functionality. Refer to the `example.py` and `demo_runner.py` source files for an overview.
 
-To reproduce the benchmark table from above run `examples/benchmark.py --scene /path/to/mp3d/17DRP5sb8fy/17DRP5sb8fy.glb`.
+To reproduce the benchmark table from above run `examples/benchmark.py --scene /path/to/mp3d_test/17DRP5sb8fy/17DRP5sb8fy.glb`.
```

Comment:
```suggestion
To reproduce the benchmark table from above run `examples/benchmark.py --scene /path/to/mp3d_example/17DRP5sb8fy/17DRP5sb8fy.glb`.
```

Please use example instead of test for examples.  This is already somewhat confusing with the habitat-test-scenes naming, but will be even more confusing with MP3D as an actual test split exists.

Reply:
Also this script will try to download the test_scene if it's not where we tell it is. May want to make sure that logic is affected.